pomerium/ctrl/bootstrap.go

```diff
@@ -66,6 +67,14 @@ func applyAuthenticate(_ context.Context, dst *config.Options, src *model.Config
 	return nil
 }
 
+func applyRuntimeFlags(_ context.Context, dst *config.Options, src *model.Config) error {
+	for k, v := range src.Spec.RuntimeFlags {
```

Comment:
if this is the only way runtime flags can be set when using the ingress controller, we should revert any runtime flags not present in src.Spec.RuntimeFlags back to the default, so that if you remove an entry in the CR the value will return to how it was before that entry was added.

Reply:
oh thats a very good point